flocx_market/cmd/api.py

```diff
@@ -1,16 +1,44 @@
+<<<<<<< HEAD
+import os
+
+from flask import Flask, request
+from flask_restful import Api
+
+from resources.offer import Offer, OfferList
+from oslo_context import context
+import esi_leap.conf
+=======
 from wsgiref import simple_server
 from flocx_market.api import app
+>>>>>>> 5e3222c8... Flake 8 tests TG-57
```

Comment:
It looks like you committed a file with some merge conflict markers. You'll want to watch out for this.